cmd/ignited/ignited.go

```diff
@@ -2,19 +2,47 @@ package main
 
 import (
 	"os"
+	"os/signal"
+	"syscall"
 
+	log "github.com/sirupsen/logrus"
 	"github.com/weaveworks/ignite/cmd/ignited/cmd"
 	"github.com/weaveworks/ignite/pkg/providers"
 	"github.com/weaveworks/ignite/pkg/providers/ignited"
 	"github.com/weaveworks/ignite/pkg/util"
 )
 
 func main() {
+	sigChan := make(chan os.Signal, 1)
+	signal.Notify(sigChan, syscall.SIGINT, syscall.SIGTERM)
+	go func() {
+		<-sigChan
+
+		log.Debug("Signal SIGTERM caught")
```

Comment:
We should either include both SIGTERM and SIGINT in the message or neither.

Reply:
I fixed this by putting name of the caught signal there in the debug log.
Thank @palemtnrider